client/src/components/Questions/Questions.jsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import QuestionsMain from './QuestionsMain';
 
-const Questions = () => (
+const Questions = ({currentProduct}) => (
   <div data-testid="questions" className="questions">
     Questions/Answers widget...
-    <QuestionsMain />
+    <QuestionsMain
+      // currentProduct={currentProduct}
+      currentProduct={currentProduct}
+    />
   </div>
 );
 
```

Comment:
I am not convinced that this file is necessary. It seems redundant with "QuestionsMain"